test/distributed/_shard/sharded_tensor/ops/test_matrix_ops.py

```diff
@@ -258,6 +223,62 @@ def test_sharded_tensor_view_error(self):
             ):
                 st.view(5, 7, -1, -1)
 
+    @with_comms(init_rpc=True)
+    @skip_if_lt_x_gpu(TEST_GPU_NUM)
+    @requires_nccl()
+    def test_sharded_tensor_layer_norm(self):
+        specs = _chunk_sharding_specs_list_for_test([1, 2], seed=10)
+        for spec in specs:
+            tensor = torch.rand(16, 35, 26).cuda(self.rank)
+            layer_norm = torch.nn.LayerNorm((35, 26)).cuda(self.rank)
```

Comment:
Can we add a test with `elementwise_affine`?

Reply:
Sure.